src/components/radiobutton/radio-button-group/radio-button-group.directive.ts

```diff
@@ -95,10 +95,28 @@ export class RadioButtonGroupDirective<T = any> implements ControlValueAccessor,
         this.onTouched();
     }
 
+    /** Determine and set the correct tabindex */
+    determineAndSetTabIndex() {
+
+        const firstEnabled = this._radioButtons.find(radio => {
+            return radio.disabled === false;
+        });
+
+        this._radioButtons.forEach(radio => {
+
+            if (this._value) {
```

Comment:
This isn't a good way to check for an undefined user-provided value, someone might need to use zero or null or an empty string as a radio option. This same issue was reported very recently for the select component, so make sure to keep a note of recurring problems like this.